src/components/sidebar/index.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState } from 'react';
+import { styled } from '@mui/material/styles';
+import {
+  Box,
+  Drawer,
+  CssBaseline,
+  List,
+  Divider,
+  IconButton,
+  ListItem,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+} from '@mui/material';
+import { Menu, ChevronLeft } from '@mui/icons-material';
+import {
+  GridOnOutlined,
+  DescriptionOutlined,
+  PlagiarismOutlined,
+  LaptopWindowsOutlined,
+  ScreenSearchDesktopOutlined,
+  SettingsOutlined,
+  PersonOutlineOutlined,
+} from '@mui/icons-material';
+
+const drawerWidth = 240;
```

Comment:
PP: constants should always be uppercase and use underline naming: "DRAWER_WIDTH"